packages/ui/components/main-nav.tsx

```diff
@@ -0,0 +1,41 @@
+import { cn } from "@kampus/ui-next/utils";
+import Link from "next/link";
+import { type PropsWithChildren, type ReactNode } from "react";
+
+interface Props {
+  brand: ReactNode;
+  links?: ReactNode;
+}
+
+export const MainNav = (props: Props) => {
```

Comment:
I'm not sure if there's alignmen on this usage or not but I'd suggest something like this to keep ability to import properly.
Just a small nitpick I'd use destructing on props if it's used in the component. This will help you to put default values.
```suggestion
export interface MainNavProps {
  brand: ReactNode;
  links?: ReactNode;
}

export const MainNav = ({brand,links}: MainNavProps) => {
```

Reply:
I am ok with renaming it to `MainNavProps`, but why export something you'll never use outside of this file? I am ok with exporting types from `@kampus/ui` packages as we might wanna extend those, but i don't think a component inside an app, that we know that's not gonna be extended, needs to export its types.

Curious, what's the thought process behind exporting it? Have you seen something that made you think that exporting it at the beginning is better than exporting it when we need to? My experience has been mostly the otherwise.

Reply:
With exporting by default gives you immediate types if it's required to use hence the rename so you know which component has which props so you can type it if you need to generate the props dynamically instead using in the render and pass all the props there. And this is type in UI package not application component. Agreed on the not exporting the type if it's app component(there might be times that is needed as well but not a big deal)

It's not an immediate benefit, just for to have consistency across development. Since types are excluded from build it does not hurt anything in that sense. I wanted to leave as a thought so this can be addressed or discussed later.